### PR TITLE
add 35 stocks from euronext

### PR DESCRIPTION
**Pull request by yngshan**

### CODE REVIEW

File: share/underlyings.yml [hunk 1]
```diff
@@ -2853,6 +2853,286 @@ frxXPTUSD:
...
+    delta_style: spot_delta
+  pip_size: 0.01
+FPBNP:
+  display_name: BNP Paribas SA
+  exchange_name: EURONEXT
+  market: stocks
+  market_convention:
+    delta_premium_adjusted: 0
+    delta_style: spot_delta
+  pip_size: 0.01
```
Comment (yngshan) [inline share/underlyings.yml]: Duplicate FPBNP , please remove one

File: share/underlyings.yml [hunk 2]
```diff
@@ -2853,6 +2853,286 @@ frxXPTUSD:
...
+FPOS:
```
Comment (yngshan) [inline share/underlyings.yml]: Please double check this . It should be FPOS or FPCS?

File: share/underlyings.yml [hunk 3]
```diff
@@ -2853,6 +2853,286 @@ frxXPTUSD:
...
+FPSAN:
+  display_name: SANFI
+  exchange_name: EURONEXT
+  market: stocks
+  market_convention:
+    delta_premium_adjusted: 0
+    delta_style: spot_delta
+  pip_size: 0.01
+FPFP:
+  display_name: TOTAL SA
```
Comment (jeff-binary) [inline share/underlyings.yml]: Lets remove all instances of SA, PLC, etc.  Makes the drop down list cleaner.

File: share/underlyings.yml [hunk 4]
```diff
@@ -2853,6 +2853,286 @@ frxXPTUSD:
...
+FPOR:
+  display_name: L'Oreal SA
+  exchange_name: EURONEXT
+  market: stocks
+  market_convention:
+    delta_premium_adjusted: 0
+    delta_style: spot_delta
+  pip_size: 0.01
+FPMC:
+  display_name: LVMH Moet Hennessy Louis Vuitton SA
```
Comment (jeff-binary) [inline share/underlyings.yml]: Lets shorten this one to Monet Hennessy Louis Vuitton.  If it doesn't fit still then we can think of something else.

File: share/underlyings.yml [hunk 5]
```diff
@@ -2853,6 +2853,286 @@ frxXPTUSD:
...
+FPRNO:
+  display_name: Renault SA
+  exchange_name: EURONEXT
+  market: stocks
+  market_convention:
+    delta_premium_adjusted: 0
+    delta_style: spot_delta
+  pip_size: 0.01
+FPKER:
+  display_name: Keiring
```
Comment (yngshan) [inline share/underlyings.yml]: According to bloomberg, the name of this company should be Kering. Please confirm

File: share/underlyings.yml [hunk 6]
```diff
@@ -2853,6 +2853,286 @@ frxXPTUSD:
     delta_style: spot_delta
     rr: call-put
   pip_size: 0.01
+FPSAN:
+  display_name: SANFI
```
Comment (yngshan) [inline share/underlyings.yml]: Sanofi or Sanfi ?

File: share/underlyings.yml [hunk 7]
```diff
@@ -2853,6 +2853,286 @@ frxXPTUSD:
...
+  pip_size: 0.01
+FPSU:
+  display_name: Schneider Electric SE
+  exchange_name: EURONEXT
+  market: stocks
+  market_convention:
+    delta_premium_adjusted: 0
+    delta_style: spot_delta
+  pip_size: 0.01
+FPAL:
```
Comment (yngshan) [inline share/underlyings.yml]: FPAI or FPAL ?

**Pull request merged**